plugins/module_utils/omevv_utils/omevv_firmware_utils.py

```diff
@@ -56,6 +65,17 @@ def get_firmware_repository_profile(self, profile_name=None):
                 profile_info = self.search_profile_name(profile_info, profile_name)
         return profile_info
 
+    def get_all_repository_profiles(self):
+        """
+        Retrieves the firmware repository profile information.
+
+        Returns:
+            list: The list of all firmware repository profile information.
+        """
+        resp = self.omevv.invoke_request('GET', PROFILE_URI)
+
+        return resp
```

Comment:
please return json_data. Applies to the places where you are returning responses.

otherwise has to do resp.json_data each time.

Reply:
this util file is used in other modules also, if I modify then other functionalities will break.